### ISSUE TITLE
#1626227349------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1081]
<strong>Source:</strong>`<button aria-label="" class="content--slotted icon-start-empty icon-end-empty" disabled="" tabindex="-1" type="button"><span class="content"><slot></slot></span></button>`

<strong>Element Location: </strong>["wmx-work-job-card:nth-child(15) > .c-jobCard > .c-jobCard__rightContainer--fixed.ng-star-inserted > .c-jobCard__actions > .c-jobCard__completeBtn[data-name=\"completeBtn\"][aria-label=\"complete\"]",".content--slotted.icon-start-empty"]

<strong>Impact: </strong>critical

<strong>Failure Summary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/button-name?application=axeAPI
